src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -988,6 +988,10 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                 load_balancer_profile=load_balancer_profile,
                 outbound_type=outbound_type,
             )
+        if load_balancer_sku.lower() == "basic":
+            network_profile = ContainerServiceNetworkProfile(
+                load_balancer_sku=load_balancer_sku.lower(),
```

Comment:
please verify whether you need to set network_plugin="kubenet" like L986

Reply:
Verified in live test that network_plugin is not needed, it default to kubenet.